salt/preflight/defaults.yaml

```diff
@@ -0,0 +1,32 @@
+destructive: False      # define if /var/lib/etcd if should be removed
+upgrade: False          # define if we're on an update case
+mandatory:
+  packages:
+    - util-linux            # provides nsenter, mount
+    - iproute               # provides ip
+    - iptables              # provides iptables
+  sysctl_values:
+    net.bridge.bridge-nf-call-ip6tables: 1
+    net.bridge.bridge-nf-call-iptables: 1
+    net.ipv4.ip_forward: 1
+  available_pathdirs:
+    - /var/lib/etcd
+    - /etc/kubernetes/manifest
+  services:
+    running:
+      - containerd
+  ports:
+    - 10250
+    - 10251
+    - 10252
+    - 2379
```

Comment:
A bit surprised this doesn't need 2380 to be open as well?

Reply:
Ah I missed it, it was not listed here: https://kubernetes.io/docs/reference/setup-tools/kubeadm/implementation-details/#kubeadm-init-workflow-internal-design but when I run the kubeadm init in verbose mode, I saw that it checks this port so I will add it